api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -106,6 +106,24 @@ enum DiscoverResource {
   DISCOVER_RESOURCE_DATABASE_POSTGRES_GCP = 15;
   DISCOVER_RESOURCE_DATABASE_MYSQL_GCP = 16;
   DISCOVER_RESOURCE_DATABASE_SQLSERVER_GCP = 17;
+
+  DISCOVER_RESOURCE_DATABASE_POSTGRES_REDSHIFT_SERVERLESS = 18;
+  DISCOVER_RESOURCE_DATABASE_POSTGRES_AZURE = 19;
+  DISCOVER_RESOURCE_DATABASE_DYNAMODB = 20;
+  DISCOVER_RESOURCE_DATABASE_CASSANDRA_KEYSPACES = 21;
+  DISCOVER_RESOURCE_DATABASE_CASSANDRA_SELF_HOSTED = 22; // Cassandra & ScyllaDb
+  DISCOVER_RESOURCE_DATABASE_ELASTICSEARCH_SELF_HOSTED = 23;
+  DISCOVER_RESOURCE_DATABASE_REDIS_ELASTICACHE = 24; // Elasticache & MemoryDb
```

Comment:
i was also confused on how we determined if we should combine two db types into one, like for example we don't separate `MySQL & MariaDB`, they are both `mysql`. 

I combined db types based on the already defined protocols here: https://github.com/gravitational/teleport/blob/337bc1c5cb58a39f1684ebf064bd78a068760e4a/lib/defaults/defaults.go#L417

Reply:
Looks good :+1: